x-pack/plugins/siem/public/components/page/hosts/first_last_seen_host/index.test.tsx

```diff
@@ -162,7 +158,7 @@ describe('FirstLastSeen Component', async () => {
         </MockedProvider>
       </TestProviders>
     );
-    await wait();
+    await wait(10);
```

Comment:
Hopefully this is the highest we have to set this. I do not mind if you do not want to not test async based actions that have wait's in them and instead we eventually rely on e2e testing.

IMO, unit tests are best when they run deterministically with no side effects in them but if this works out with just `10` reliably that is ok with me.

Reply:
++ 

This should hopefully go away with my upcoming PR as there are changes to how the `FirstLastSeen` component manages error reporting which shouldn't require any additional wait time. 🤞 Either way, definitely agree keeping unit tests deterministic is _extremely_ important. I think there may be one or two of the new super datepicker tests that are not, so will have to keep an eye out on them.